app/views/pages/help.html.slim

```diff
@@ -2,80 +2,193 @@
+
+p = t('pages.help.signing_in.answer_6_part_1')
+
+p = t('pages.help.signing_in.answer_6_part_2')
+
+h3 = t('pages.help.signing_in.question_7')
+
+p = t('pages.help.signing_in.answer_7_part_1')
+
+p = t('pages.help.signing_in.answer_7_part_2')
+
+p = t('pages.help.signing_in.answer_7_part_3')
+
+h2 = t('pages.help.changing_settings.title')
+
+h3 = t('pages.help.changing_settings.question_1')
+
+p = t('pages.help.changing_settings.answer_1_part_1_html', app: link_to(APP_NAME, root_url))
+
+p = t('pages.help.changing_settings.answer_1_part_2')
+
+h3 = t('pages.help.changing_settings.question_2')
+
+p = t('pages.help.changing_settings.answer_2', app: APP_NAME)
+
+h3 = t('pages.help.changing_settings.question_3')
+
+p = t('pages.help.changing_settings.answer_3', app: APP_NAME)
+
+h2 = t('pages.help.privacy_and_security.title')
+
+h3 = t('pages.help.privacy_and_security.question_1', app: APP_NAME)
+
+p = t('pages.help.privacy_and_security.answer_1_part_1', app: APP_NAME)
+
+p = t('pages.help.privacy_and_security.answer_1_part_2', app: APP_NAME)
+
+p = t('pages.help.privacy_and_security.answer_1_part_3_html', nist_link: link_to(t('pages.help.privacy_and_security.nist_link'), 'https://www.nist.gov/'))
+
+p = t('pages.help.privacy_and_security.answer_1_part_4_html',
+  info_at_login_mailto: link_to('info@login.gov', 'mailto:info@login.gov'), security_privacy_policy_link: link_to(t('pages.help.privacy_and_security.security_privacy_policy_link'), privacy_path))
+
+h3 = t('pages.help.privacy_and_security.question_2', app: APP_NAME)
+
+p = t('pages.help.privacy_and_security.answer_2', app: APP_NAME)
+
+h3 = t('pages.help.privacy_and_security.question_3')
+
+p = t('pages.help.privacy_and_security.answer_3_part_1')
+
+p = t('pages.help.privacy_and_security.answer_3_part_2')
+
+p = t('pages.help.privacy_and_security.answer_3_part_3')
+
+p = t('pages.help.privacy_and_security.answer_3_part_4')
+
+h3 = t('pages.help.privacy_and_security.question_4')
+
+p = t('pages.help.privacy_and_security.answer_4_part_1', app: APP_NAME)
+
+ul
+
+  li = link_to(t('pages.help.privacy_and_security.chrome'), 'https://support.google.com/chrome/answer/95606')
+
+  li = link_to(t('pages.help.privacy_and_security.firefox'), 'https://support.mozilla.org/kb/password-manager-remember-delete-change-passwords#w_viewing-and-deleting-passwords')
+
+  li = link_to(t('pages.help.privacy_and_security.internet_explorer'), 'https://support.microsoft.com/en-us/help/17499/windows-internet-explorer-11-remember-passwords-fill-out-web-forms#ie=ie-11')
+
+  li = link_to(t('pages.help.privacy_and_security.safari'), 'http://help.apple.com/safari/mac/8.0/#/ibrw1103')
+
+p = t('pages.help.privacy_and_security.answer_4_part_2')
+
+h3 = t('pages.help.privacy_and_security.question_5')
+
+p = t('pages.help.privacy_and_security.answer_5_part_1')
+
+p = t('pages.help.privacy_and_security.answer_5_part_2_html', info_at_login_mailto: link_to('info@login.gov', 'mailto:info@login.gov'), app: APP_NAME)
```

Comment:
ooh mail_to TIL! and Figaro....I would tend towards not because it's such a hassle to manage env vars right now, and I don't expect this to change often?